src/data_structures/src/bit_array.cairo

```diff
@@ -1,20 +1,20 @@
-use array::{serialize_array_helper, deserialize_array_helper};
-use byte_array::BYTES_IN_BYTES31_MINUS_ONE;
-use bytes_31::{
-    one_shift_left_bytes_u128, one_shift_left_bytes_felt252, bytes31, BYTES_IN_U128,
-    BYTES_IN_BYTES31,
-};
-use integer::u512;
-use serde::into_felt252_based::SerdeImpl;
+use core::bytes_31::{bytes31};
+use core::integer::u512;
+use core::serde::Serde;
+use core::serde::into_felt252_based::SerdeImpl;
 
 const SELECT_BIT: u128 = 0b10;
+const POW_2_128: felt252 = 0x100000000000000000000000000000000;
+const BYTES_IN_U128: usize = 16;
+const BYTES_IN_BYTES31: usize = 31;
+const BYTES_IN_BYTES31_MINUS_ONE: usize = BYTES_IN_BYTES31 - 1;
```

Comment:
not `pub`?

Reply:
Cause not use anywhere, but could do, lmk